src/utils/calendar.js

```diff
@@ -0,0 +1,113 @@
+const listOfMonth = [
+  "January",
+  "February",
+  "March",
+  "April",
+  "May",
+  "June",
+  "July",
+  "August",
+  "September",
+  "October",
+  "November",
+  "December",
+]
+
+const listOfDays = [
```

Comment:
is it possible for locale (`listOfDays`, `listOfMonth`) to be defined using `fecha`?. :thinking:
so, any kind of date formatting should refer to `utils/date-input`, to ensure consistent locale/language.

Reply:
I saw design in figma doesn't have options formatting to date locale, the date format fixed with token pattern 'MM/DD/YYYY'.

So i think doesn't need refer to `utils/date-input` because there is not have relation

Reply:
i see i see, thank you for your response :kiss: